### ISSUE TITLE
Complete Sustainable Clothing Model

### ISSUE THREAD

**Issue by colleenskemp** [ProjectDrawdown/solutions#289]
The initial import of this model has been done, but it produces errors.  The task is now to complete the implementation.
An NDA is required to access the Excel

**Comment by olawale1rty** [comment]
Nice, working on this issue.

**Comment by nelsonkhcheng** [comment]
Hi Nelson here, I'm on the team for this issue in the hackathon!

**Comment by nelsonkhcheng** [comment]
Work on this new model is still a work in progress.
I am looking to check this into a new branch "hackathon-sustainableclothing" and will also look to merge it to the hackathon branch as per Dev's suggestion

Here is the progress made so far:

Created /sustainableclothing extraction model folder and Sustainable_Clothing.ipynb Notebook based of Extraction_Guide
Created test_SustainableClothing_RRS() Test function in Test_excel_integration.py
Running the test gave AssertionErrors on data comparison.
Investigation of errors focused on First Cost data
Investigations and attempts to fix errors documented in Sustainable_Clothing_FirstCostChecks.ipynb, which informed "solution/scenario specific hack fixes" made in /solution/sustainableclothing/__init__.py

Solution specific fix 1) the 2014 World First Cost value not matching between model and test data.
Basically, in the Excel, in Helper Tables, the data for 2014 in C91 is pulling from cell C85, which ultimately seems to be an sourced from an Adoption for 2018 number, which is set in the Tab Advanced Controls in C61. 
This didn't seem to be happening in the code, so in __init__.py, we just hardcoded/overwrite this value in the model data

Solution specific error 2) This issue we weren't able to fix. I added a similar hardcoded fix attempt in __init__.py just for one scenario, but this hadn't fixed the error. 
In the model, the "New implementation units required (includes replacement units)" data for 2018 seems to have that same Adoption for 2018 value in it from the Tab Advanced Controls in C61, rather than the actual 2018 number from the Excel file. We tried to hardcode overwrite this value with the value from the Excel file for Scenario Dec 2020, but it didn't seem to work, I'm guessing the data is overwritten in a function in one of the model functions.
But hopefully the team can review what we tried to do and implement a real solution!
